session.h

```diff
@@ -23,13 +23,8 @@
 #include "global.h"
 
 #ifdef WITH_CONTIKI
-#include "ip/uip.h"
-typedef struct {
-  unsigned char size;
-  uip_ipaddr_t addr;
-  unsigned short port;
-  int ifindex;
-} session_t;
+#include "coap-endpoint.h"
```

Comment:
As tinydtls is not CoAP-specific in any way, I would rather not create this dependency.

Reply:
it's true, I just corrected this adding it only in the case that it is being built in CoAP where the pointer does have to be the same.